addons/binding/org.openhab.binding.yeelight/README.md

```diff
@@ -52,4 +53,18 @@ Items:
 ```
 Color YeelightLEDColor { channel="yeelight:stripe:1:color" }
 Switch YeelightLEDSwitch { channel="yeelight:stripe:1:color" }
+String YeelightLEDCommand { channel="yeelight:stripe:1:command" }
 ```
+
+Rules:
+
+```
+rule "Yeelight Custom Command"
+when
+        Time is noon
+then
+        YeelightLEDCommand.sendCommand("set_power;\"on\",\"smooth\",2000")
+end
+```
+
+Note that `set_power` is the command, then a separator `;` must be used. `\"on\",\"smooth\",2000` are the parameters.
```

Comment:
Thanks for adding an example. Do you know, whether the command API is documented somewhere, so the users can see all available options? Then this could be linked in the Readme as well.

Reply:
The API is documented here: https://www.yeelight.com/download/Yeelight_Inter-Operation_Spec.pdf

Reply:
Added.